apps/api/src/domain/models/vocabulary-list/commands/translate-vocabulary-list-name/vocabulary-list-name-translated.event.ts

```diff
@@ -0,0 +1,5 @@
+import { BaseEvent } from '../../../shared/events/base-event.entity';
+import { VOCABULARY_LIST_TRANSLATED } from './constants';
+export class VocabularyListTranslated extends BaseEvent {
+    readonly type = VOCABULARY_LIST_TRANSLATED;
```

Comment:
Change this to `VocabularyListNameTranslated` and `VOCABULARY_LIST_NAME_TRANSLATED`.

Reply:
done